plugins/woocommerce/tests/e2e-pw/test-data/data.js

```diff
@@ -51,30 +51,455 @@ const customer = {
 	},
 };
 
+const languageObject = {
+	en: {
```

Comment:
Is there a benefit to naming the language objects in `lang-COUNTRY` format like `en-US` or `en-GB` to be consistent with the values acceptable to Playwright's [`locale`](https://playwright.dev/docs/api/class-testoptions#test-options-locale) config option?

Reply:
Love this suggestion - thanks!